client/src/components/screens/PatientSearch/index.jsx

```diff
@@ -461,12 +459,11 @@ class PatientSearchScreen extends React.Component {
     }));
 
     return (
-      <Content>
-        <Header />
+      <Layout>
         <Card className="patientSearch">
           <Row>
             <Col span={24}>
-              <Title level={3}>{ intl.get('screen.patientsearch.title') }</Title>
+              <Title level={3}>{intl.get('screen.patientsearch.title')}</Title>
```

Comment:
I still have some small formatting changes but those fits with the remaining of the code base. Might have been commited with wrong formatting in the first place?